website/source/docs/auth/token.html.md

```diff
@@ -286,8 +286,10 @@ of the header should be "X-Vault-Token" and the value should be the token.
       <li>
         <span class="param">increment</span>
         <span class="param-flags">optional</span>
-            An optional requested lease increment can be provided. This
-            increment may be ignored.
+            An optional requested lease increment (measured in seconds) can be provided.
+            If changing the token's TTL to current time + `increment` seconds would
```

Comment:
Ok, shall I just link to the overall token concepts page rather than that section of it?